### PR TITLE
Move roles to CONTRIBUTING.rst

### LINKED ISSUE #10184: Move relevant information about Roles to CONTRIBUTING.rst
**Description**

The old documentation for [Roles](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=62694605) should be reviewed and incorporated in [CONTRIBUTING.rst](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst)

**Use case / motivation**

To remove documentation duplication and clean it up.

**Related Issues**

#10178

### PR DESCRIPTION
**Pull request by jpmarques66**
Adds [roles](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=62694605) to [CONTRIBUTING.rst](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst) as requested in #10184 .
Also fixes a typo in breeze command.

Closes #10184 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -17,6 +17,70 @@
...
+Additionally, committers are listed in a few other places (some of these may only be visible to existing committers):
+
+* https://whimsy.apache.org/roster/ppmc/airflow
+* https://github.com/orgs/apache/teams/airflow-committers/members
+
+Committers are responsible for:
+
+* Championing one or more items on the `Roadmap <https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Home>`__
+* Reviewing & Merging Pull-Requests
+* Scanning and responding to JIRAs
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
* Scanning and responding to Github issues
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -17,6 +17,70 @@
...
+* Responding to questions on the dev mailing list (dev@airflow.incubator.apache.org)
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
* Responding to questions on the dev mailing list (dev@airflow.apache.org)
```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -17,6 +17,70 @@
...
+Becoming a Committer
+--------------------
+
+There is no strict protocol for becoming a committer.
+Candidates for new committers are typically people that are active contributors and community members.
+
+The key aspects of a committer are:
+
+* Consistent contributions over the past 6 months
+* Expressed understanding of Airflow Core or has displayed holistic understanding of a particular part and made
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
* Understanding of Airflow Core or has displayed a holistic understanding of a particular part and made
```

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -17,6 +17,70 @@
...
+  contributions towards a more strategic goal
+* Understands contributor/committer guidelines: `Contributors' Guide <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst>`__
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
* Understanding of contributor/committer guidelines: `Contributors' Guide <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst>`__
```

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -17,6 +17,70 @@
...
+* Quality of the commits, and asses the impact of the changes
```
Comment (turbaszek) [inline CONTRIBUTING.rst]: What does it mean `asses the impact of the changes`?

File: CONTRIBUTING.rst [hunk 6]
```diff
@@ -17,6 +17,70 @@
...
+* Visibility on discussions on the dev mailing list and the Slack channel
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
* Visibility in community discussions (on the dev mailing list, Slack and Github)
```
WDYT?

File: CONTRIBUTING.rst [hunk 7]
```diff
@@ -17,6 +17,70 @@
...
+* Testing Release Candidates to improve the release cycle
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
* Testing Release Candidates
```

File: CONTRIBUTING.rst [hunk 8]
```diff
@@ -154,7 +219,7 @@ You can use the default Breeze configuration as follows:
 
 .. code-block:: bash
 
-   ./breeze --initialize-local-virtualenv
+   ./breeze initialize-local-virtualenv
```
Comment (turbaszek) [inline CONTRIBUTING.rst:221]: Is this a related change?

File: CONTRIBUTING.rst (hunk 1 above)
Comment (potiuk) [reply CONTRIBUTING.rst]: +1

File: CONTRIBUTING.rst (hunk 2 above)
Comment (potiuk) [reply CONTRIBUTING.rst]: +1

File: CONTRIBUTING.rst (hunk 3 above)
Comment (potiuk) [reply CONTRIBUTING.rst]: yep

File: CONTRIBUTING.rst (hunk 4 above)
Comment (potiuk) [reply CONTRIBUTING.rst]: yep

File: CONTRIBUTING.rst (hunk 5 above)
Comment (potiuk) [inline CONTRIBUTING.rst]:
```suggestion
* Quality of the commits

File: CONTRIBUTING.rst (hunk 6 above)
Comment (potiuk) [reply CONTRIBUTING.rst]: +1

File: CONTRIBUTING.rst (hunk 7 above)
Comment (potiuk) [reply CONTRIBUTING.rst]: +1

File: CONTRIBUTING.rst (hunk 8 above)
Comment (potiuk) [reply CONTRIBUTING.rst:221]: I think not related but I would not obiect fixing it.

**Review by potiuk — COMMENTED** [review]
Few clarifications - mostly agree with @turbaszek !

File: CONTRIBUTING.rst (hunk 1 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: Done.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: done.

File: CONTRIBUTING.rst (hunk 3 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: done.

File: CONTRIBUTING.rst (hunk 4 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: done.

File: CONTRIBUTING.rst (hunk 5 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: As @potiuk suggested, I removed that part.

File: CONTRIBUTING.rst (hunk 6 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: @turbaszek Agreed.

File: CONTRIBUTING.rst (hunk 7 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: done.

File: CONTRIBUTING.rst (hunk 8 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst:221]: Not related, just a typo I noticed when trying to use Breeze for the first time. If no one opposes, I'll keep it here.

File: CONTRIBUTING.rst (hunk 5 above)
Comment (jpmarques66) [reply CONTRIBUTING.rst]: Removed.

**Review by turbaszek — APPROVED** [review]
LGTM! 🚀 Just a question: should this be the first point in CONTRIBUTING.rst? @potiuk @kaxil ?

**Comment by potiuk** [comment]
Good point @turbaszek . I think it should be seond one - "Contributing" should be first. WDYT @kaxil  ?

**Review by kaxil — COMMENTED** [review]
Yeah definitely 2nd. :) Don't want the new contributors to have an information overload

**Review by ryw — APPROVED** [review]

File: CONTRIBUTING.rst [hunk 9]
```diff
@@ -84,6 +84,70 @@ If you are proposing a new feature:
...
+Considering Apache, the role of the PMC is primarily to ensure that Airflow conforms to Apache's processes
+and guidelines.
+
+Committers/Maintainers
+----------------------
+
+Committers are community members that have write access to the project’s repositories, i.e., they can modify the code,
+documentation, and website by themselves and also accept other contributions.
+
+The official list of committers can be found `here <https://airflow.apache.org/docs/>`__.
```
Comment (kaxil) [inline CONTRIBUTING.rst]:
```suggestion
The official list of committers can be found `here <https://airflow.apache.org/docs/stable/project.html#committers>`__.
```

**Review by potiuk — APPROVED** [review]

**Comment by jpmarques66** [comment]
@kaxil thanks for the contribution 🙏. Should we merge this?

**Comment by potiuk** [comment]
Some checks were cancelled I think - it's best if you rebase on latest master and re-push @jpmarques66 -> then we can merge it with confidence :)

**Comment by jpmarques66** [comment]
@potiuk I think it looks good now. Thanks!

**Comment by potiuk** [comment]
Cool! Thanks !

**Comment by kaxil** [comment]
Thanks @jpmarques66

**Comment by potiuk** [comment]
Thanks @jpmarques66 ! One step closer to clean docs ! Looking forward to next contributions!
